contracts/governance/Governor.sol

```diff
@@ -25,12 +26,15 @@ import "./IGovernor.sol";
  *
  * _Available since v4.3._
  */
-abstract contract Governor is Context, ERC165, EIP712, IGovernor, IERC721Receiver, IERC1155Receiver {
+abstract contract Governor is Context, ERC165, EIP712, Nonces, IGovernor, IERC721Receiver, IERC1155Receiver {
     using DoubleEndedQueue for DoubleEndedQueue.Bytes32Deque;
 
-    bytes32 public constant BALLOT_TYPEHASH = keccak256("Ballot(uint256 proposalId,uint8 support)");
+    bytes32 public constant BALLOT_TYPEHASH =
+        keccak256("Ballot(uint256 proposalId,uint8 support,address voter,uint256 nonce)");
```

Comment:
Do we ave any rational for ordering of these ? My intuition would be to put voter before support. I can't explain why, and it honestly doesn't change much ... but still I think we may want to set some guidelines.

Reply:
I see value in setting a guideline for this, but the main reason for this order is that it's the same order in which I got the parameters of `castVoteWithReasonAndParamsBySig` to work. Other orderings don't compile due to `Stack too deep`.

I actually might consider using a struct memory but wanted to avoid that discussion for now :sweat_smile:

Reply:
wow ok. That is both a terrible reason, and a very practical one :P

Reply:
It's sort of a deadline in a sense 🧠 

By the way, I removed the `currentNonce` variable as suggested so maybe there's now room for reordering these but I still have no criteria. I'd invoke @frangio again for an opinion (if any)

![imagen(5)](https://github.com/OpenZeppelin/openzeppelin-contracts/assets/33379285/fe2f898e-da45-4ce8-9fde-181272ebb94b)

Reply:
I don't think ordering is important so I would leave as is.